pkg/controllers/gacha-controller.go

```diff
@@ -120,3 +121,44 @@ func UpdateUser(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 
 }
+
+func CreateCharacter(w http.ResponseWriter, r *http.Request) {
```

Comment:
CreateCharacter API is not defined in swagger.yaml.
What do you need it for?

Reply:
To add the characters, we can also add it directly through the mysql `insert` statement as well. I created it to make request directly from postman and view it.

Reply:
Okay I will remove this api function